xformers/components/attention/csrc/cuda/mem_eff_attention/kernel_forward.h

```diff
@@ -239,9 +239,9 @@ struct AttentionKernel {
             scalar_t, // ElementC
             accum_t // ElementAccumulator
             >;
-    static constexpr int64_t kAlignmentA =
+    static constexpr int kAlignmentA =
```

Comment:
Was this change only to make it closer to cutlass conventions?

Reply:
Avoided a conversion from `int64_t` (`long int`) -> `int` when passing through cutlass's templates